### ISSUE TITLE
v0.2.0 keeps sending endless scroll event, no scroll happening

### ISSUE THREAD

**Issue by Eldzej** [darkroomengineering/lenis#18]
Lenis version: 0.2.0
Browsers: All
OS: Windows

In v0.2.0, Lenis seem to have some kind of a problem, and keep sending endless scroll events, which causes scroll to not happen (as it keep trying to go back up to 0, I assume)

Minimal demo, try scrolling, both mousewheel and scrollbar, and check the console for the endless event being sent:
https://codepen.io/eldzej/pen/oNdXVgR

Been using 0.1.12 to great satisfaction after a ton of issues with LocomotiveScroll, so not sure what happen in v0.2.0

**Comment by clementroche** [comment]
Thank you, it will be fixed asap

**Comment by clementroche** [comment]
fixed in v0.2.1, now you must pass time to `lenis.raf()`

```js
function raf(time) {
  lenis.raf(time)
  requestAnimationFrame(raf)
}

requestAnimationFrame(raf)
```

See [codepen](https://codepen.io/ClementRoche/pen/YzLXMQx?editors=1010)
